ocs_ci/utility/utils.py

```diff
@@ -2262,6 +2247,27 @@ def ceph_health_check_base(namespace=None):
         raise CephHealthException(f"Ceph cluster health is not OK. Health: {health}")
 
 
+def wait_for_rook_ceph_tools_pod(namespace, timeout=300):
```

Comment:
Is this function necessary? Shouldn't this be covered already in:
wait_for_rook_ceph_tools_pod 
?

Reply:
Sorry, I didn't get the comment. Do you say that this function already exists somewhere? Or do we not need it?

Reply:
Sorry, wrong copy paste, meant: `get_ceph_tools_pod`. Updated the comment.

Reply:
We don't wait for the `rook-ceph-tool` pod to be ready in this function. But we may add this option.